exposed-tests/src/test/kotlin/org/jetbrains/exposed/sql/tests/shared/entities/CompositeIdTableEntityTest.kt

```diff
@@ -0,0 +1,198 @@
+package org.jetbrains.exposed.sql.tests.shared.entities
+
+import org.jetbrains.exposed.dao.CompositeEntity
+import org.jetbrains.exposed.dao.CompositeEntityClass
+import org.jetbrains.exposed.dao.id.CompositeID
+import org.jetbrains.exposed.dao.id.CompositeIdTable
+import org.jetbrains.exposed.dao.id.EntityID
+import org.jetbrains.exposed.sql.*
+import org.jetbrains.exposed.sql.tests.DatabaseTestsBase
+import org.jetbrains.exposed.sql.tests.TestDB
+import org.jetbrains.exposed.sql.tests.currentTestDB
+import org.jetbrains.exposed.sql.tests.shared.assertEquals
+import org.jetbrains.exposed.sql.tests.shared.expectException
+import org.junit.Test
+import java.util.UUID
+import kotlin.test.assertIs
+import kotlin.test.assertNotNull
+import kotlin.test.assertTrue
+
+// SQLite excluded from all tests as it only allows auto-increment on single column PKs.
+// SQL Server is sometimes excluded because it doesn't allow inserting explicit values for identity columns.
+class CompositeIdTableEntityTest : DatabaseTestsBase() {
+    object Publishers : CompositeIdTable("publishers") {
```

Comment:
I'm thinking if we can try make CompositeIdTable to IdTable by default, let's discuss